.github/workflows/make-protos.yml

```diff
@@ -0,0 +1,39 @@
+name: Generate attestation libraries
+on:
+  push:
+    branches:
+      - "main"
+  schedule:
+    - cron: '0 14 * * 1'
+jobs:
+  run-make-protos:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install protoc
+        uses: arduino/setup-protoc@ab6203da1c3118e4406048171b09238ad31ad73e
+        with:
+          version: '3.x'
+      - name: Install Go
+        uses: actions/setup-go@4d34df0c2316fe8122ab82dc22947d607c0c91f9
+        with:
+          go-version: 1.20.x
+      - name: setup env
+        run: |
+          echo "$(go env GOPATH)/bin" >> $GITHUB_PATH
+      - name: Checkout repo
+        uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab
+      - name: Generate libraries
+        run: |
+          make protos
+      - name: Create PR
+        uses: peter-evans/create-pull-request@5b4a9f6a9e2af26e5f02351490b90d01eb8ec1e5
```

Comment:
Is it secure to use this (random guy's?) thing for modifying code?  My guess is no one is going to provide much review on these PRs since they're machine generated.

Reply:
It's like the arduino thing above, it's a random guy but it's a pretty popular action for the job (https://github.com/peter-evans/create-pull-request/). Note that the action is only creating a commit and PR if the prior step `make protos` resulted in something to commit.